content/en/docs/components/pipelines/overview/multi-user.md

```diff
@@ -1,175 +1,179 @@
-            type=ApiResourceType.NAMESPACE,
-        ),
-        relationship=ApiRelationship.OWNER,
-    )],
-))
+import kfp
+from kfp_server_api import *
+
+# the namespace in which you deployed Kubeflow Pipelines
+kubeflow_namespace = "kubeflow"
+
+# the namespace of your pipelines user (where the pipeline will be executed)
+user_namespace = "jane-doe"
+
+# when `path` is None, the `KF_PIPELINES_SA_TOKEN_PATH` environment variable is used
+# (if `KF_PIPELINES_SA_TOKEN_PATH` is unset, the default is: `/var/run/secrets/kubeflow/pipelines/token`)
+credentials = kfp.auth.ServiceAccountTokenVolumeCredentials(path=None)
+
+# create a client
+client = kfp.Client(host=f"http://ml-pipeline-ui.{kubeflow_namespace}", credentials=credentials)
+
+# create an experiment
+experiment: ApiExperiment = client._experiment_api.create_experiment(
+    body=ApiExperiment(
+        name="<YOUR_EXPERIMENT_ID>",
+        resource_references=[
+            ApiResourceReference(
+                key=ApiResourceKey(
+                    id=user_namespace,
+                    type=ApiResourceType.NAMESPACE,
+                ),
+                relationship=ApiRelationship.OWNER,
+            )
+        ],
+    )
+)
+print("-------- BEGIN: EXPERIMENT --------")
 print(experiment)
-pipeline = client.pipelines.list_pipelines().pipelines[0]
-print(pipeline)
-client.runs.create_run(body=ApiRun(
-    name='test-run-1234',
-    pipeline_spec=ApiPipelineSpec(
-        pipeline_id=pipeline.id,
-    ),
-    resource_references=[ApiResourceReference(
-        key=ApiResourceKey(
-            id=experiment.id,
-            type=ApiResourceType.EXPERIMENT,
+print("-------- END: EXPERIMENT --------")
+
+# get the experiment by name (only necessary if you comment out the `create_experiment()` call)
+# experiment: ApiExperiment = client.get_experiment(
+#     experiment_name="<YOUR_EXPERIMENT_ID>",
+#     namespace=user_namespace
+# )
+
+# create a pipeline run
+run: ApiRunDetail = client._run_api.create_run(
+    body=ApiRun(
+        name="<YOUR_RUN_NAME>",
+        pipeline_spec=ApiPipelineSpec(
+            # replace <YOUR_PIPELINE_ID> with the UID of a pipeline definition you have previously uploaded
+            pipeline_id="<YOUR_PIPELINE_ID>",
         ),
-        relationship=ApiRelationship.OWNER,
-    )],
-))
-runs=client.runs.list_runs(
+        resource_references=[ApiResourceReference(
+            key=ApiResourceKey(
+                id=experiment.id,
+                type=ApiResourceType.EXPERIMENT,
+            ),
+            relationship=ApiRelationship.OWNER,
+        )
+        ],
+    )
+)
+print("-------- BEGIN: RUN --------")
+print(run)
+print("-------- END: RUN --------")
+
+# view the pipeline run
+runs: ApiListRunsResponse = client._run_api.list_runs(
     resource_reference_key_type=ApiResourceType.EXPERIMENT,
     resource_reference_key_id=experiment.id,
 )
+print("-------- BEGIN: RUNS --------")
 print(runs)
+print("-------- END: RUN --------")
```

Comment:
NIT: `END: RUNS` instead of `END: RUNS`

Reply:
Resolved in https://github.com/kubeflow/website/pull/3021/commits/c3bf6da6cbc8e517174b1d91cdbf9e034464eb6e